src/frontend/window/main_window.rs

```diff
@@ -42,6 +42,8 @@ impl SimpleComponent for AppModel {
 
         // we use this window only as a container for the screenshot windows
         window.hide();
```

Comment:
```suggestion
```
You already asked it on my PR why not using `hide`: [It's deprecated](https://gtk-rs.org/gtk4-rs/stable/latest/docs/gtk4/prelude/trait.WidgetExt.html#method.hide).